virtc/main.go

```diff
@@ -100,6 +100,12 @@ var podConfigFlags = []cli.Flag{
 		Value: "",
 		Usage: "the hyperstart tty socket type",
 	},
+
+	cli.StringSliceFlag{
+		Name:  "volume",
+		Value: new(cli.StringSlice),
+		Usage: "the volume to be shared with VM",
+	},
```

Comment:
Here you should declare that one as a `cli.GenericFlag` and then get it directly as a `Volume`. See how virtc/main.go does that for `agentType` for example.